ledger-service/db-backend/src/main/scala/com/digitalasset/http/dbbackend/Queries.scala

```diff
@@ -782,10 +842,29 @@ private object OracleQueries extends Queries {
     }
   }
 
+  // XXX as with containsAtContractPath, literalScalar is too big a type
```

Comment:
xxx? Is that a todo? If so, let’s please use TODO so we only have to grep for one thing.

Reply:
I use it as a sort of weaker TODO that shouldn't block anything (i.e. in some respect not finding it with grep is a feature). Is there a better way to express that in a lightweight way?

Reply:
not sure, I tend to use NOTE and TODO but NOTE mostly for things that are purely informational and shouldn’t be acted upon. Fine to keep XXX as well if you want.